nimbus.fml.yaml

```
@@ -0,0 +1,30 @@
+channels:
+  - debug
+  - nightly
+  - beta
+  - release
+features:
+  tabs:
+    description: Nimbus feature name intended to control the multiple tabs feature in the app.
+    variables:
+      is_multi_tab:
+        description: Nimbus variable of [FEATURE_TABS] allowing outside control of whether the multiple tabs feature should be enabled or not.
+        type: Boolean
+        default: true
+  onboarding-with-cfrs:
+    description: Nimbus feature name intended to control the onboarding plus all CFRs in the app.
+    variables:
+      is-onboarding-with-cfrs:
+        description: If `true`, the app will show the new onboarding screen and all the cfrs
+        type: Boolean
+        default: false
+  onboarding-without-cfrs:
+    description: Nimbus feature name intended to control the onboarding without CFRs in the app.
+    variables:
+      is-onboarding-without-cfrs:
+        description: If `true`, the app will show only the new onboarding screen
+        type: Boolean
+        default: false
+types:
+  objects: {}
+  enums: {}
```

`types` is redundant.

```suggestion
objects: {}
enums: {}
```